pkg/loop/lggr_broker.go

```diff
@@ -51,3 +105,24 @@ type resource struct {
 	io.Closer
 	name string
 }
+
+type resources []resource
```

Comment:
where is resource defn? can we put this and the following funcs next to it?

Reply:
It is right there ^ if you expand.